subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -186,15 +237,16 @@ class Libero {
         git.checkout(branch: config.releaseName)
         mavenResult = maven.execute(buildRequest)
         if (mavenResult.getExitCode() != 0) {
+            log.error("Release Build Failed due to: ${mavenResult.executionException}")
             throw new IllegalStateException( "Release Build failed!!" );
         }
-        // TODO: dry run should either remove tags and reset, or provide advice on how to do so
         if (!options.dryRun) {
```

Comment:
I forgot a couple of times in the past to use the proper `settings.xml` file when running `libero` and my artifacts would fail to upload. This is very frustrating because there's no way to restart from that point.

Do you think we could add a warning when the user doesn't use the dry-run option telling them to double check that they used the right settings file? Or even better, test that the upload will work before getting started? ;-)